apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -65,6 +70,20 @@ const OrganizationFooter: React.FC<HeaderProps> = ({ organizationPage }) => {
   }
 }
 
+const OrganizationChatPanel = ({ slug }: { slug: string }) => {
+  // remove when organization chat-bot is ready for release
+  if (process.env.NODE_ENV !== 'development') {
+    return null
+  }
+
+  switch (slug) {
+    case 'syslumenn':
+      return <ChatPanel endpoint="syslumenn" />
+    default:
+      return null
+  }
+}
```

Comment:
Consider making the `slug === endpointKey` so `endpoints[slug]` resolves to correct config, we might not need this abstraction in if that is the case

Reply:
I tried doing something like this
```
slug in endpoints ? ​<ChatPanel endpoint={slug} /> : null
```
But TS kept complaining, so I did it this way.